bundle/manifests/wlo-controller-manager_v1_serviceaccount.yaml

```diff
@@ -1,9 +0,0 @@
-apiVersion: v1
```

Comment:
I believe we had to add this in the first place to change the service account name in CSV (wlo-controller-manager as opposed to controller-manager). I am guessing that the prefix `wlo-` is automatically added by Operator-SDK to Service Account?